src/Makefile.am

```diff
@@ -432,6 +431,7 @@ libbitcoin_util_a_SOURCES = \
   logging.cpp \
   random.cpp \
   rpc/protocol.cpp \
+  shutdown.cpp \
```

Comment:
Moving `shutdown.cpp` to util might not be a good choice, architecturally: it's shared between *all* executables, by far most of which don't use bitcond's init/shutdown mechanism. It's meant for general-purpose utilities.

`server` would be the right place, it's really part of the server functionality (shared between `bitcoind` and `bitcoin-qt` and unit tests…).

Reply:
@laanwj Unfortunately, I can't move random to server, because rpc/protocol is in util and depends on random.

I think I'll just leave the 10ms strengthening uninterruptible instead?

Reply:
I can wait 10ms